### ISSUE TITLE
terminal command lines not working on Windows 10 Anaconda 

### ISSUE THREAD

**Issue by knslee07** [practical-nlp/practical-nlp#13]
## Location

https://github.com/practical-nlp/practical-nlp/blob/master/Ch3/05_Pre_Trained_Word_Embeddings.ipynb
https://github.com/practical-nlp/practical-nlp/blob/master/Ch4/01_OnePipeline_ManyClassifiers.ipynb

## Current Behavior

'wget' is not recognized as an internal or external command, operable program or batch file.
'apt-get' is not recognized as an internal or external command, operable program or batch file.

## Steps to Reproduce

1. !pip install wget
2. run the code, then the error.
3. !pip install apt-get, then "ERROR: Could not find a version that satisfies the requirement apt-get (from versions: none)
ERROR: No matching distribution found for apt-get"
4. rune the code, then "'apt-get' is not recognized as an internal or external command, operable program or batch file."
5. More detailed explanation in the original message below.

## Context (Environment)

running on Jupyter Notebook
Python 3.8.3
Windows 10 64, Conda, both base and virtual env (python=3.6)
Clean installed Windows and Anaconda.

## Possible Implementation

Found the template and added the above. The below is the original message in writing.
Fyi, I am not a coder nor data scientist so the error could be due to my lack of knowledge.

Hello,

I have been enjoying this book since the moment I bought this last week and trying to walk through the commands on Jupyter.

I am on Windows 10 64-bit, not Mac and having trouble executing lines after !. They do not work on my desktop (e.g., Ch. 3-5, wget, Ch. 4-1 apt-get). I first googled and saw ! does not work on Windows but others say bash commands are now okay. So, I installed wget-3.2 (!pip install wget) on Jupyter but still unable to to execute it. I get the following message: "'wget' is not recognized as an internal or external command, operable program or batch file." I even installed a separate program wget on Windows because others on the internet say it has to be a path variable.
Also, I could not even install apt-get (using !pip install apt-get) with the following message "ERROR: Could not find a version that satisfies the requirement apt-get (from versions: none) ERROR: No matching distribution found for apt-get". 

Could you help me with these and possibly other terminal commands in the following chapters? Given the growing popularity of NLP and the excellent quality of this book, I am sure more people on Windows 10 will walk through the commands here and might face the same issues.

P.S. I also think the TF-IDF table in Ch. 3 is hard to understand based on the definition you provided.

**Comment by suranah** [comment]
Thank you for your comment, Prof. Lee. We will try to get back to you soon on this.

**Comment by knslee07** [comment]
Thank you suranah.
I did my homework a bit more.
Both wget and apt-get are linux line commands for downloading and installing. Even though I installed wget for Windows, bash commands in Jupyter did not work. For wget, I worked around this by running the line (!wget ...) on cmd or Power Shell. For some reason, however, the command itself was not sufficient and added "--no-check-certificate". After getting data on cmd, I came back to Jupyter and ran the rest of the lines.
For apt-get, it does not seem to be possible to use that command on Windows. There are alternatives like oneget or winget, chocolatey. For platform-neutral scripts or for Windows users, other alternative codes may be needed that use oneget or winget or choco. This should apply for all the other codes in the following chapters.

**Comment by varunp2k** [comment]
Prof. Lee. We are working to fix this issue [here](https://github.com/practical-nlp/practical-nlp/issues/15). A temporary fix would be to use the python wget package.
`!pip install wget`
`import wget`
`wget.download(Download URL)`
In the case of Ch4 [notebook](https://github.com/practical-nlp/practical-nlp/blob/master/Ch4/01_OnePipeline_ManyClassifiers.ipynb) it will be:
`wget.download('https://raw.githubusercontent.com/practical-nlp/practical-nlp/master/Ch4/Data/Full-Economic-News-DFE-839861.csv')`
Then you can set the path appropriately.

Apologies for the inconvenience caused.  We are actively working fixing this and making the experience as smooth as possible for all users.

Regards,
PracticalNLP Team

**Comment by suranah** [comment]
Thanks again for opening this issue, Prof. Lee. We are working on making sure the experience is as seamless as possible on Windows. 

As our book is now being adopted as the official textbook for universities in Virginia, Texas & Germany, we have made a longer roadmap for 2020 to fix a range of issues that have been raised. You can read the roadmap here:

https://github.com/practical-nlp/practical-nlp/issues/14

Would love your feedback on it. Thank you!

**Comment by knslee07** [comment]
Thank you for your reply Harshit. Good to hear that the book has been adopted at several colleges. 
One way to get issues around is WSL. I was able to avoid most of the issues by installing WSL on my Windows and Anaconda on WSL. Your scripts work way better on Linux thanks to the similarity between Mac Terminal and Linux Bash.
Still, there are quite a few issues like dependency. Many modules are imported without being first installed. It would be no problem on your Mac, but first-time nlp reader will not be able to run the scripts. Don't know exactly why but it is harder to use terminal command in Jupyter on Windows (even with admin status), so I had to get out of Jupyter and installed modules on Terminal and got back. Also, there are issues of 'pip install' and 'anaconda install'. I suggest you clean install Anaconda or clean Python and run the codes in Jupyter notebook on Windows,

Another issue I found was about topic modeling in Ch. 7. The data used for the demonstration is located in the local path: data_path = "/home/etherealenvy/Downloads/booksummaries/booksummaries.txt". There is no way for others to follow the notebook.
